### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by bytestream**
|      Q       |   A
|------------- | -----------
| Type         | improvement
| BC Break     | no
| Fixed issues | 

#### Summary

Exclude `ci` from Composer

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -13,3 +13,4 @@
 /tests export-ignore
 /.travis.yml export-ignore
 /UPGRADE.md export-ignore
+/ci export-ignore
```
Comment (morozov) [inline .gitattributes:16]: Please use alphabetical order (before `/composer.lock`).

**Review by morozov — CHANGES_REQUESTED** [review]

File: .gitattributes (hunk 1 above)
Comment (bytestream) [reply .gitattributes]: Sure. I've updated it.

**Review by morozov — APPROVED** [review]

**Comment by morozov** [comment]
@greg0ire I recall there was an attempt to only include certain files into the package as opposed to constantly excluding the new ones. Do you remember how it ended?

**Comment by morozov** [comment]
Thanks, @bytestream.

**Pull request merged**